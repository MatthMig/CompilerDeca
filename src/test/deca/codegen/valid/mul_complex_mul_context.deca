// Description :
//
//      Test contextuellement correct d'une multiplication  complexe.
// Résultats : 
// 
//      N/A
//
// Historique :
// 
//      Créé le 07/06/2023 par gl03

{
    int i=1;
    while (i<99)
    {
        i=i*1;
    }
    int y=((((2*3)))*((4*5))*6*(7*8))*9;
    i=i*y;
    println(y);
    println(i);
}