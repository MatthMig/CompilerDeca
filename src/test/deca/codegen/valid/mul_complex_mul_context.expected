362880
23224320